processing/src/test/java/io/druid/query/extraction/RegexDimExtractionFnTest.java

```diff
@@ -117,13 +117,54 @@ public void testNullAndEmpty()
   }
 
   @Test
+  public void testMissingValueReplacement()
+  {
+    String regex = "(a\\w*)";
+    ExtractionFn extractionFn = new RegexDimExtractionFn(regex, true, "foobar", false);
+    Set<String> extracted = Sets.newHashSet();
+
+    for (String testString : testStrings) {
+      extracted.add(extractionFn.apply(testString));
+    }
+
+    Assert.assertEquals(4, extracted.size());
+    Assert.assertTrue(extracted.contains("apple"));
```

Comment:
@b-slim I'm following the style of the existing tests, but I can go ahead and change the test style within the file.